alvr/client_core/src/connection.rs

```diff
@@ -90,6 +94,7 @@ pub fn connection_lifecycle_loop(
     ctx: Arc<ConnectionContext>,
     lifecycle_state: Arc<RwLock<LifecycleState>>,
     event_queue: Arc<Mutex<VecDeque<ClientCoreEvent>>>,
+    headset_is_worn: Arc<AtomicBool>,
```

Comment:
Why is the AtomicBool wrapped in an arc, it shouldn't need that, you can just give it a reference